lib/ifupdown.sh.in

```diff
@@ -16,209 +16,209 @@ fi
 . '@utilsfuncfile@'
 
 case "$IF_MSTPCTL_PORTS" in
-    "")
+    '')
 	exit 0
 	;;
     none)
-	INTERFACES=""
+	INTERFACES=''
 	;;
     *)
 	INTERFACES="$IF_MSTPCTL_PORTS"
 	;;
 esac
 
 # Previous work (create the interface)
-if [ "$MODE" = "start" ] && [ ! -d /sys/class/net/$IFACE ]; then
-  brctl addbr $IFACE || exit 1
+if [ "$MODE" = 'start' ] && [ ! -d "/sys/class/net/$IFACE" ]; then
+  brctl addbr "$IFACE" || exit 1
 # Previous work (stop the interface)
-elif [ "$MODE" = "stop" ]; then
-  ifconfig $IFACE down || exit 1
+elif [ "$MODE" = 'stop' ]; then
+  ifconfig "$IFACE" down || exit 1
 fi
 
 all_interfaces= &&
 unset all_interfaces &&
 mstpctl_parse_ports $INTERFACES | while read i; do
```

Comment:
shellcheck.com says you missed quotes on the `$INTERFACES` var

it also says other stuff, but this is the only one that fits for this commit